src/app/watermelon-mountain-ranch/watermelon-mountain-ranch.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="container-fluid mt-4 pt-4">
+  <h1 class="text-center text-info font-weight-bold">
+    Watermelon Mountain Ranch
+  </h1>
+  <br />
+  <!--JUMBOTRON-->
+  <div class="container-fluid">
+    <div class="jumbotron jumbotron-fluid rounded p-5">
+      <ngb-carousel>
+        <ng-template ngbSlide>
+          <div class="row">
+            <div class="col-9">
+              <a
+                href="watermelon-mountain-ranch"
+                alt="Watermelon Mountain Ranch"
+                title="Watermelon Mountain Ranch"
+              >
+                <img
+                  src="assets/puppyadoption.svg"
+                  alt="Watermelon Mountain Ranch"
+                  class="rounded mx-auto"
+                />
+              </a>
+            </div>
+            <div class="col-3">
+              <div class="card">
+                <h3 class="font-weight-bold text-light">
+                  Watermelon Mountain Ranch
+                </h3>
+                <p class="d-lg-none d-xl-block text-light">
+                  Puppy adoption event 8/31 <br />9:00am-3:00pm <br />@ Petsmart
+                  on Coors Blvd.
+                </p>
+              </div>
+            </div>
+          </div>
+        </ng-template>
+        <ng-template ngbSlide>
+          <div class="row">
+            <div class="col-9">
+              <a href="watermelon-mountain-ranch">
+                <img
+                  src="assets/puppyadoption.svg"
+                  alt="Watermelon Mountain Ranch"
+                  class="rounded mx-auto"
+                />
+              </a>
+            </div>
+            <div class="col-3">
+              <div class="card">
+                <h3 class="font-weight-bold text-light">
+                  Watermelon Mountain Ranch
+                </h3>
+                <p class="d-lg-none d-xl-block text-light">
+                  Puppy adoption event 8/31 <br />9:00am-3:00pm <br />@ Petsmart
+                  on Coors Blvd.
+                </p>
+              </div>
+            </div>
+          </div>
+        </ng-template>
+        <ng-template ngbSlide>
+          <div class="row">
+            <div class="col-9">
+              <a href="watermelon-mountain-ranch">
+                <img
+                  src="assets/puppyadoption.svg"
+                  alt="Watermelon Mountain Ranch"
+                  class="rounded mx-auto"
+                />
+              </a>
+            </div>
+            <div class="col-3">
+              <div class="card">
+                <h3 class="font-weight-bold text-light">
```

Comment:
I would discourage you from the class names like this for styling. They should instead intend to identify the element, not the style you want to adopt. (You can make use of constants for persistent styling values, too.)
http://bdavidxyz.com/blog/how-to-name-css-classes/

Reply:
Jk bootstrap. Forgot that was a thing.